osschallenge/templates/osschallenge/projectindex.html

```diff
@@ -6,13 +6,14 @@
     {% if project_list %}
         <ul class="index">
         {% for project in project_list %}
-
+        {% if project.active is True %}
```

Comment:
Like this, you're still loading all the projects from the database, and only filter them in the view. It might be better to filter already on the database level?